packages/core-sdk/.env.example

```diff
@@ -1,2 +1,4 @@
 TEST_WALLET_ADDRESS = 0x0000000000000000000000000000000000000000
-WALLET_PRIVATE_KEY = 0x0000000000000000000000000000000000000000
\ No newline at end of file
+WALLET_PRIVATE_KEY = 0x0000000000000000000000000000000000000000
+JUDGE_PRIVATE_KEY = 0x0000000000000000000000000000000000000000
```

Comment:
i think we should think about other ways to handle this. Since it seems this key cannot just be any private key that people can create. This will make it hard for someone to set this up and verify tests locally. One option is separate the tests that require this as its own integration tests that should only be run by CI and not meant to be run like the other integration tests locally.

Reply:
Done. It will check for the `JUDGE_PRIVATE_KEY` in the workflow, if it doesn't find it, it will skip run the tests.